docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket.rst

```diff
@@ -86,25 +87,32 @@ Let us assume your bucket is called `datalake`. And let us assume the folder you
 
         Get security credentials (access key id, secret access key) from AWS
 
-**Preparing your data**
-
+Preparing your data
+^^^^^^^^^^^^^^^^^^^^^
 
 For Lightly to be able to create embeddings and extract metadata from your data, `lightly-magic` needs to be able to access your data. You can either download/sync your data from S3 or you can mount S3 as a drive. We recommend downloading your data from S3 as it makes the overall process faster.
```

Comment:
maybe also add a link to `:ref:`lightly-command-line-tool`.` similar as done in the glcoud tutorial